DataFormats/L1TMuonPhase2/src/classes_def.xml

```diff
@@ -1,27 +1,36 @@
 <lcgdict>
- <selection>
+    <selection>
+        <class name="l1t::MuonStub" ClassVersion="3">
```

Comment:
This seems a format change. Please avoid making changes to the GMT part if not needed, as I am preparing another PR on this.

Reply:
This was necesary to include the new data structures for EMTF++.

Reply:
I see. That is OK.